submissions/ALT+F4 LTN-RLD-Cybersyn Mod Comparison/Requesting a Train.md

```diff
@@ -0,0 +1,115 @@
+image:
+title:
+lang: en
+discuss:
+  discord: https://alt-f4.blog/discord
+  forum: https://forums.factorio.com/viewtopic.php?f=5&t=98585
+  reddit: https://www.reddit.com/r/factorio/comments/nmxfxn/altf4_37_combinator_crash_course_continuation/
+---
+
+## Mod Spotlight / Mod Comparison / Requesting A Train <author>OceanPhantom</author>
+
+Factorio's train system is great. Trains are the transportation method of choice when it comes to long distances. Its rules are exceedingly simple, allowing you to build your first automated bi-directional rail network to a nearby ore outpost with incredible ease. And yet these rules, taken all together, can make setting up a network of hundreds of trains to feed your megabases's insatiable demand for raw resources a logistic nightmare. 
+
+They allow for such incredible complexity that [u/minibetrayal](https://www.reddit.com/u/minibetrayal/) on reddit theorized that the train system was actually Turing Complete. And then took on the challenge of [building the largest and most useless rail network ever](https://www.reddit.com/r/factorio/comments/evkff6/the_biggest_and_most_useless_rail_network_ever/) to prove it. Making the rail system join the circuit network and the belt transport system ([Alt-F4 #23](https://alt-f4.blog/ALTF4-23/) and [Alt-F4 #24](https://alt-f4.blog/ALTF4-24/)) as yet another independently Turing Complete part of Factorio.
+
+With all the options available to us, how could the train system possibly get better? 
+
+Enter the Factorio modding community. 
+
+We've talked before about how mods have improved the rail system. They introduced the first iteration of [the fluid wagon](https://alt-f4.blog/ALTF4-21/). They've changed trains' behaviour, with mods like [Train Supply Manager (TSM)](https://mods.factorio.com/mod/train-pubsub), which we've talked about previously, all the way back in [Alt-F4 #15](https://alt-f4.blog/ALTF4-15/). 
+
+But it came to my attention that while ALT+F4 has touched on it briefly in several separate previous articles ([Alt-F4 #28](https://alt-f4.blog/ALTF4-28/), [Alt-F4 #37](https://alt-f4.blog/ALTF4-37/)), there has yet to be one that takes an in-depth look at the [Logistic Train Network](https://mods.factorio.com/mod/LogisticTrainNetwork) mod (also known as LTN for short), and what it does. Until recently, LTN was unique, with the only other notable mod that altered how trains functioned being the previously mentioned TSM.
+
+That all changed in November 2022, with the advent of not one, but two new competitors, which unlike TSM, directly compete with LTN's style of Logistics. With two new competitors, [Project Cybersyn](https://mods.factorio.com/mod/cybersyn) and [Rail Logistics Dispatcher](https://mods.factorio.com/mod/RailLogisticsDispatcher/changelog) being recently released, now seemed like a perfect time to compare and contrast of all three mods.
+
+{% include image.html src='' alt='Modded Entities' caption='The entities added by all three mods' %}
```

Comment:
can you write a caption that describes the image in a way that a screen reader would give the person a good concept of what they're "looking" at? :pray:

Reply:
The alt text can be the same as the regular caption. If it's not good enough we can fix it later ourselves, no worries.